build/github/build-and-publish-probes-for-operating-system/main.go

```diff
@@ -191,6 +210,48 @@ func getFalcoDrivers(dockerCli *docker.Client, FalcoVersionNames []string) ([]fa
 	return FalcoVersions, nil
 }
 
+func falcoVersionHasRelease(releases []*github.RepositoryRelease, falcoVersions []falcoVersion) bool {
+	for _, fv := range falcoVersions {
+		for _, r := range releases {
+			if strings.HasPrefix(fv.Driver, r.GetName()) {
+				return true
+			}
+		}
+	}
+
+	return false
+}
+
+func kernelPackageHasBeenCompiled(releases []*github.RepositoryRelease, kernelPackage string) bool {
```

Comment:
nit: same with this, could we whack this in `//pkg/repository/ghreleases`? We can extend the `pkg/repository.Repository` interface if we need :)